fastplotlib/graphics/image.py

```diff
@@ -393,3 +394,44 @@ def add_linear_region_selector(
         selector.offset = selector.offset + (0.0, 0.0, self.offset[-1] + 1)
 
         return selector
+
+    def add_rectangle_selector(
+        self,
+        selection: tuple[float, float, float, float] = None,
+        fill_color=(0, 0, 0.35, 0.2),
```

Comment:
```suggestion
```

Reply:
I think for images we want to keep this kwarg because the alpha value needs to be 0.2 which is different than the default for adding a rectangle selector for lines or line collections